### PR TITLE
[WIP] Fix Autobahn test failures

### PR DESCRIPTION
**Pull request by mikaelm12**
There was a failure in the Autobahn tests because they were searching for a file that no longer exists.

### CODE REVIEW

**Comment by Tratcher** [comment]
Doesn't this need to go into 2.1 first and then get merged forward?

File: src/Middleware/WebSockets/test/ConformanceTests/Helpers.cs [hunk 1]
```diff
@@ -8,21 +8,8 @@ public class Helpers
...
-                {
-                    return Path.GetFullPath(Path.Combine(directoryInfo.FullName, "test", projectName));
-                }
-
-                directoryInfo = directoryInfo.Parent;
-            }
-            while (directoryInfo.Parent != null);
-
-            throw new Exception($"Solution root could not be found using {applicationBasePath}");
+            return Path.GetFullPath(Path.Combine(directoryInfo.FullName, "test", projectName));
```
Comment (natemcmaster) [inline src/Middleware/WebSockets/test/ConformanceTests/Helpers.cs]:
```suggestion
            return Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, projectName));
```
I don't think "test" is going to appear in the path anymore, and you can get rid of the directoryInfo variable.

**Comment by mikaelm12** [comment]
> Doesn't this need to go into 2.1 first and then get merged forward?

Yes, you're right

**Comment by mikaelm12** [comment]
Replacing in favor of https://github.com/aspnet/AspNetCore/pull/4302

**Pull request closed without merging**